governance-exercises/Le Bateleur - Poll Exercise 02/Course of Action.md

```diff
@@ -0,0 +1,61 @@
+# Poll Exercise 02
+
+---
+
+## Course of Action
+
+### Summary
+
+In the Exercise Prompt, I interpret "course of action" to encompass the immediate steps of responding to the Forum thread and other pre-drafting needs.
+
+Assuming no further evidence concerning Clive's identity is provided, my course of action is as follows:
+
+1. Given my mandate to minimize risk to Maker and guard against misalignment threats, I interpret Diane's materials as rising to the level of "significant suspicion" specified in Atlas 2.6.6.
+
+2. Before derecognition can be finalized, however, the Atlas requires a "check" on my decision in the form of a review from FacilitatorDAOs. FacilitatorDAOs do not yet exist. So, in my discretion as a Governance Facilitator (and to go with the flow of the Exercise), I run a governance poll which can serve as this "check." The poll: "Express your support or opposition to the determination that the breach-of-anonymity evidence pertaining to Aligned Delegate Clive rises to the level of 'significant suspicion,' as specified in Atlas 2.6.6."
+
+   If the governance poll comes out against this determination, I would consider the matter resolved in favor of Clive. If the governance poll comes out in favor of this determination, I would derecognize Clive.
+
+3. I respond on the Forum to explain my course of action and respond to Clive and Diane's posts.
+
+---
+
+### Poll on derecognition
+
+This Exercise concerns derecognition of Alignment Conservers for breach-of-anonymity and the associated whistleblower bounty. But protocols around derecognition and the whistleblower bounty have not yet been established. To resolve the situation, I have to make use of powers of interpretation and discretion. [GOV 2.7.6](https://mips.makerdao.com/mips/details/MIP101#2-7-6-decision-making-powers-of-facilitatordaos) authorizes FacilitatorDAOs to "make interpretations and take discretionary decisions based on the language of the Alignment Artifacts." In the Pregame, Governance Facilitators act in the place of FacilitatorDAOs. (See [1](https://mips.makerdao.com/mips/details/MIP113#7-facilitatordaos-and-facilitators), [2](https://mips.makerdao.com/mips/details/MIP113#7-1-active-facilitators), [3](http://forum.makerdao.com/t/governance-scope-actions-pertaining-to-delegates-in-light-of-mip102c2-sp7-approval/20946).) 
+
+Two instances of governance practice precedent exist. (See [1](http://forum.makerdao.com/t/governance-scope-actions-pertaining-to-delegates-in-light-of-mip102c2-sp7-approval/20946), [2](http://forum.makerdao.com/t/notice-aligned-delegate-derecognition-and-avc-member-warning/21099).) Both times, the Governance Facilitator unilaterally acted to derecognize the Aligned Delegates in question; no Governance Poll was involved. In both instances, the identity/whereabouts of the Aligned Delegates was confirmed or strongly suspected; and the Aligned Delegates did not dispute the derecognition process. 
```

Comment:
I think I told you this in advance, which in retrospect I probably shouldn't have. But yeah, it is more or less implied by how those cases were handled, given nobody disputed publicly on the forum.